app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,18 @@
+class SessionsController < ApplicationController
+  def new
+  end
+
+  def create
+    if user = User.from_omniauth(request.env["omniauth.auth"])
+      session[:user_id] = user.id
+      redirect_to root_path
+    else
+      redirect_to root_path
```

Comment:
we will probably need to redirect this somewhere else, or the else is kinda redundant